frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -91,23 +97,27 @@ function Resource({
                   >
                      {title}
                   </Link>
-                  <Prerendered
-                     lineHeight="1.36"
-                     fontWeight="regular"
-                     fontSize="12px"
-                     color="gray.600"
-                     overflow="hidden"
-                     height="16px"
-                     noOfLines={1}
-                     html={introduction}
-                  />
+                  {introduction && (
+                     <Prerendered
+                        lineHeight="1.36"
+                        fontWeight="regular"
+                        fontSize="12px"
+                        color="gray.600"
+                        overflow="hidden"
+                        height="16px"
+                        noOfLines={1}
+                        html={introduction}
+                     />
+                  )}
                </Stack>
                <Stack
                   direction="row"
                   justify="flex-start"
                   align="flex-start"
-                  spacing="4px"
+                  spacing="0px"
                   alignSelf="stretch"
+                  flexWrap="wrap"
+                  gridGap="4px"
```

Comment:
Is this a related feature to chakra `gap`? I can't find anything online about `gridGap`. I'm concerned that it also might not work with older versions of safari like `gap`.

Reply:
`gridGap` corresponds to `grid-gap`, which is an older name for the `gap` property. I think we were using `display: grid` along with `gap` at one point for the guide flags before we switched to HStack with the scroll instead of wrap. Either way this runs into issues with older Safari since this is still a `display: flex`.

Reply:
[90b05ba](https://github.com/iFixit/react-commerce/pull/1599/commits/90b05ba9dc56e8c1cc0d0a7b7be5bda77d7f4c49)